### ISSUE TITLE
Support Information Functions for MySQL

### ISSUE THREAD

**Issue by tristaZero** [apache/shardingsphere#2235]
### Which version of ShardingSphere did you use?
4.0.0.0-RC1
### Which project did you use? Sharding-JDBC or Sharding-Proxy?
Sharding-Proxy
### Expected behavior
1. input `select database();`
2. return current schema name;
### Actual behavior
1. input `select database();`
2. return actual data node name, e.g. 
```
mysql> select database();
+-----------------+
| database()      |
+-----------------+
| demo_ds_slave_0 |
+-----------------+
```

**Comment by Moriadry** [comment]
Hi miss, I am wondering do you guys get a way to integration test sharding-proxy, I don't know how to debug this kind of bugs locally.

**Comment by tuohai666** [comment]
@Moriadry 
Download sharding-proxy from: https://www.apache.org/dyn/closer.cgi?path=incubator/shardingsphere/4.0.0-RC1/apache-shardingsphere-incubating-4.0.0-RC1-sharding-proxy-bin.tar.gz
You can start this binary package locally. Don't forget to copy mysql-connector-java-5.1.47 to lib directory manually.

**Comment by Moriadry** [comment]
Get it.

**Comment by terrymanu** [comment]
FYI: https://dev.mysql.com/doc/refman/8.0/en/information-functions.html

**Comment by totalo** [comment]
Hello, I want to try to do this. I want to know whether we aggregate the data of these Informations or choose a table to display. How do we distinguish nodes from different sources when we aggregate?

**Comment by sandynz** [comment]
In 5.0.0-RC1, test result:
```
mysql> select database();
+-------------+
| DATABASE()  |
+-------------+
| sharding_db |
+-------------+
```

It's already implemented in  #8539, related code is `ShowCurrentDatabaseExecutor`:
```
    public void execute(final BackendConnection backendConnection) {
        mergedResult = new SingleLocalDataMergedResult(Collections.singleton(backendConnection.getSchemaName()));
    }
```
It return current connection related schema name.
